en/index.md

```diff
@@ -378,27 +378,31 @@ This will work, until such a time as whoever owns the image closes their account
 ## Words and phrases to avoid
 
 Try to avoid jargon, long-winded phrases and words with negative
-connotations. Steer clear of the following:
-
- - *Allow* - This suggests that we are in a position of power,
-permitting users or customers to conduct certain activities.
- - *The ability to* – Use 'We can' instead of 'We have the ability to'
- - *Is able to* – Use 'Ubuntu can' instead of 'Ubuntu is able to'
- - *Not only...but also...*
- - *Eliminate*
- - *Execute*
- - *Terminate*
- - *Kill*
- - *Disruptive*
- - *Explosive*
- - *Leverage*
- - *Ecosystem*
- - *Going forward*
- - *In order to*
- - *Form factor*
- - *Use case*
- - *End user* – Use 'user' instead
- - *Linux for human beings*
+connotations:
+
+| Avoided              | Motivation              | Substitution examples               |
+|----------------------|-------------------------|-------------------------------------|
+|Allow                 | This suggests that we are in a position of power, permitting users or customers to conduct certain activities.|                          |
+|The ability to        |                         | Use 'We can' instead of 'We have the ability to'                        |
+|Is able to            |                         | Use 'Ubuntu can' instead of 'Ubuntu is able to' |
+|Not only...but also...|                         |                               |
+|Eliminate             |                         |                               |
+|Execute               |                         |                               |
+|Terminate             |                         |                               |
+|Kill                  |                         |                               |
+|Disruptive            |                         |                               |
+|Explosive             |                         |                               |
+|Leverage              |                         |                               |
+|Ecosystem             |                         |                               |
+|Going forward         |                         |                               |
+|In order to           |                         |                               |
+|Form factor           |                         |                               |
+|Use case 	           |                         |                               |
```

Comment:
The pain of tables... :) Formatting...

Nitpicking here but if we use tables we should either attempt to align everything or just stick with column size without whitespace padding.

Reply:
Well, I was trying to align it. The problem is, that it looks different (aligned) in my IDE (VS Code). I guess I need to prioritize the GitHub-rendered look.

Reply:
I fixed it. The problem was some invisible symbols that take multiple spaces when rendered, most probably tabulations.